baseline/default

```diff
@@ -5037,52 +5037,52 @@
       ]
     }
   }, 
-  "test/gst-vaapi/decode/avc.py:test_default(case=1080p)": {
+  "test/gst-vaapi/decode/avc.py:default.test(case=1080p)": {
     "md5": "2873526ec49defd754c7853f0658799d"
   }, 
-  "test/gst-vaapi/decode/avc.py:test_default(case=720p)": {
+  "test/gst-vaapi/decode/avc.py:default.test(case=720p)": {
     "md5": "e6b24a42be6b698f1bf96b75d1480ade"
   }, 
-  "test/gst-vaapi/decode/avc.py:test_default(case=QCIF)": {
+  "test/gst-vaapi/decode/avc.py:default.test(case=QCIF)": {
     "md5": "61aeb97a3ab5da00a28a962ae9d69a28"
   }, 
-  "test/gst-vaapi/decode/avc.py:test_default(case=QVGA)": {
+  "test/gst-vaapi/decode/avc.py:default.test(case=QVGA)": {
     "md5": "767f88894d9635691c6e1af82b1a738b"
   }, 
-  "test/gst-vaapi/decode/hevc.py:test_8bit(case=1080p)": {
+  "test/gst-vaapi/decode/hevc.py:default.test(case=1080p)": {
     "md5": "3f66eca6367872ffbe2d53ab0c573852"
   }, 
-  "test/gst-vaapi/decode/hevc.py:test_8bit(case=720p)": {
+  "test/gst-vaapi/decode/hevc.py:default.test(case=720p)": {
     "md5": "fc0bde09b5befabb0c17593ca51bdb37"
   }, 
-  "test/gst-vaapi/decode/hevc.py:test_8bit(case=QCIF)": {
+  "test/gst-vaapi/decode/hevc.py:default.test(case=QCIF)": {
     "md5": "c6d9e88dcbcec23942344649c0c01bc4"
   }, 
-  "test/gst-vaapi/decode/hevc.py:test_8bit(case=QVGA)": {
+  "test/gst-vaapi/decode/hevc.py:default.test(case=QVGA)": {
```

Comment:
kind remind that do not forget 10bit reference md5 value

Reply:
Thanks.  We don't have 10bit test cases in the example config.